pkgs/development/python-modules/cx_oracle/default.nix

```diff
@@ -0,0 +1,27 @@
+{ stdenv, buildPythonPackage, fetchPypi, oracle-instantclient }:
+
+buildPythonPackage rec {
+  pname = "cx_Oracle";
+  version = "6.1";
+  name = "${pname}-${version}";
+
+  buildInputs = [
+    oracle-instantclient
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "80545fc7acbdda917dd2b1604c938141256bdfed3ad464a44586c9c2f09c3004";
+  };
+  
+  checkPhase = ''
+    LD_LIBRARY_PATH=$LD_LIBRARY_PATH:/usr/lib/oracle/12.2/client64/lib
```

Comment:
:question:

Reply:
You are referring to a path outside the store. That is incorrect. Furthermore, paths to libraries are better hardcoded. Also, something should *run* in the `checkPhase`.

Reply:
Okay, Is there a good way to handle the path?